src/t8_forest/t8_forest_general.h

```diff
@@ -761,6 +761,28 @@ int
 t8_forest_element_point_inside (t8_forest_t forest, t8_locidx_t ltreeid, const t8_element_t *element,
                                 const double point[3], const double tolerance);
 
+/** Query whether a batch of point lies inside an element or not. For bilinearly interpolated elements.
+ * \note For 2D quadrilateral elements this function is only an approximation. It is correct
+ *  if the four vertices lie in the same plane, but it may produce only approximate results if 
+ *  the vertices do not lie in the same plane.
+ * \param [in]      forest      The forest.
+ * \param [in]      ltree_id    The forest local id of the tree in which the element is.
+ * \param [in]      element     The element.
+ * \param [in]      points      3-dimensional coordinates of the points to check
+ * \param [in]      num_points  The number of points to check
+ * \param [in, out] is_inside   An array of length num_points, filled with 0/1 on output. True (non-zero) if \a point 
+ *                              lies within \a element, false otherwise. The return value is also true if the point 
+ *                              lies on the element boundary. Thus, this function may return true for different leaf 
+ *                              elements, if they are neighbors and the point lies on the common boundary.
+ * \param [in]      tolerance   tolerance that we allow the point to not exactly match the element.
+ *                              If this value is larger we detect more points.
+ *                              If it is zero we probably do not detect points even if they are inside
+ *                              due to rounding errors.
```

Comment:
`\a` will not put an "a" into the documentation. "a" has to be set individually
```suggestion
/** Query whether a batch of points lies inside an element. For bilinearly interpolated elements.
 * \note For 2D quadrilateral elements this function is only an approximation. It is correct
 *  if the four vertices lie in the same plane, but it may produce only approximate results if 
 *  the vertices do not lie in the same plane.
 * \param [in]      forest      The forest.
 * \param [in]      ltree_id    The forest local id of the tree in which the element is.
 * \param [in]      element     The element.
 * \param [in]      points      3-dimensional coordinates of the points to check
 * \param [in]      num_points  The number of points to check
 * \param [in, out] is_inside   An array of length \a num_points, filled with 0/1 on output. True (non-zero) if a \a point 
 *                              lies within an \a element, false otherwise. The return value is also true if the point 
 *                              lies on the element boundary. Thus, this function may return true for different leaf 
 *                              elements, if they are neighbors and the point lies on the common boundary.
 * \param [in]      tolerance   Tolerance that we allow the point to not exactly match the element.
 *                              If this value is larger we detect more points.
 *                              If it is zero we probably do not detect points even if they are inside
 *                              due to rounding errors.
```